os_info.yaml

```diff
@@ -70,6 +76,17 @@ rhel-9:
     - aarch64
     - ppc64le
   type: rpm
+sles-12:
+  version_name: 12
+  arch:
+    - amd64
```

Comment:
is [s390x here possible](https://jira.mariadb.org/browse/MDEV-28713?focusedCommentId=225494&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-225494)?